app-admin/flannel/flannel-9999.ebuild

```diff
@@ -23,7 +23,7 @@ RDEPEND="app-admin/sdnotify-proxy"
 S="$WORKDIR"
```

Comment:
Remove `RDEPEND="app-admin/sdnotify-proxy"` since it is no longer used. Set to rkt instead.